geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseBackedMetadata.scala

```diff
@@ -28,9 +28,8 @@ class HBaseBackedMetadata[T](connection: Connection, catalog: TableName, val ser
   override protected def createTable(): Unit = {
     WithClose(connection.getAdmin) { admin =>
       if (!admin.tableExists(catalog)) {
-        val descriptor = new HTableDescriptor(catalog)
-        HBaseVersions.addFamily(descriptor, ColumnFamilyDescriptor)
-        admin.createTable(descriptor)
+        HBaseVersions.createTableAsync(admin, catalog, Seq(ColumnFamily), None, None, None, Some(true), None, Seq.empty)
```

Comment:
Do we ever use this function by itself?  I wonder if it'd be 'better' to run the 'waitForTable' together and just have a createTable method?

Reply:
could be... i'm not sure if we use it separately, but it mirrors the actual method from hbase so i think it makes sense to keep any other logic out of the reflection api.